sql/driver/conn.go

```
@@ -112,6 +109,9 @@ func (c *conn) send(args Request) (*rows, error) {
 		return r, nil
 	}
 	result := resp.Results[index]
+	if result.Error != nil {
```

This no longer silently drops an error.